changelogs/fragments/10661-support-gpg-auto-impor-keys-in-zypper.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - zypper - support the --gpg-auto-import-keys option in zypper (https://github.com/ansible-collections/community.general/issues/10660, https://github.com/ansible-collections/community.general/pull/10661).
```

Comment:
```suggestion
  - zypper - support the ``--gpg-auto-import-keys`` option in zypper (https://github.com/ansible-collections/community.general/issues/10660, https://github.com/ansible-collections/community.general/pull/10661).
```

Reply:
Thanks, this makes sense. :)
I just updated the PR.